tcms/kiwi_auth/forms.py

```diff
@@ -113,3 +115,26 @@ def save(  # pylint: disable=too-many-arguments
             html_email_template_name,
             extra_email_context,
         )
+
+
+class GroupAdminForm(forms.ModelForm):
+    class Meta:
+        model = Group
+        fields = ['name', 'permissions']
+
+    users = forms.ModelMultipleChoiceField(
+         queryset=User.objects.all(),
+         required=False,
+         widget=FilteredSelectMultiple('users', False)
+    )
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        if self.instance.pk:
+            self.fields['users'].initial = self.instance.user_set.all()
+
+    def save(self, commit=True):
+        instance = super().save(commit)
+        self.instance.user_set.set(self.cleaned_data['users'])
```

Comment:
Try adding a new group without any selected permissions and users and saving it. Raises an exception:

 "<Group: NoPerms>" needs to have a value for field "id" before this many-to-many relationship can be used.